src/main/java/com/chopshop166/chopshoplib/sensors/InvertDigitalInput.java

```diff
@@ -0,0 +1,50 @@
+package com.chopshop166.chopshoplib.sensors;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+
+/**
+ * uses DigitalInputSource interface to create a invertable input source (cuz !
```

Comment:
This should follow the style that Jeff used in the `Differential Drive Map` class.

Reply:
🗸?